tests/ndarray/test_c2array_proxy.py

```diff
@@ -0,0 +1,95 @@
+#######################################################################
```

Comment:
For consistency, I'd prefer this module to be called `test_proxy_c2array.py`

Reply:
Fixed in 42c27b8bc76bc1c4ed98a708df63fec788dd1b0a.